src/main/java/seedu/address/model/eatery/Review.java

```diff
@@ -38,12 +38,13 @@
      * @param cost Cost of the meal being reviewed.
      * @param rating Rating out of 5 for the meal being reviewed.
      */
-    public Review(String description, double cost, int rating) {
+    public Review(String description, double cost, int rating, String date) {
         requireAllNonNull(description, cost, rating);
```

Comment:
is date supposed to be optional? why is it not included in requireAllNonNull()? also, if it is supposed to be optional, shouldn't we use the date when the user adds the review? but i don't see any code that does that hahaha